downstream/modules/platform/proc-controller-configure-analytics.adoc

```diff
@@ -22,7 +22,7 @@ image::automation_analytics.png[Automation analytics page]
 * *Red Hat or Satellite password*: This password is used to send data to {Analytics}.
 * *Last gather date for Automation Analytics*: Set the date and time
 * *Automation Analytics Gather Interval*: Interval (in seconds) between data gathering.
-//No tooltip and no information for this.
-* *Last gathered entries from the data collection service of {Analytics}*: TBD
+//This field has been removed.
```

Comment:
Analytics is now enabled automatically and there is no screen or settings from within the Subscription page. Here is the wording I've used elsewhere:

When you imported your license for the first time, you were automatically opted in for the collection of data that powers {Analytics}, a cloud service that is part of the {PlatformNameShort} subscription.

To enable or disable this feature, use the following steps:

. From the navigation panels, select {MenuSetSystem}.
. Click btn:[Edit].
. Toggle the *Gather data for {Analytics}* switch and enter your Red Hat customer credentials.
. You can also configure the following options:
*Red Hat customer username*: This username is used to send data to {Analytics}.
* *Red Hat customer password*: This password is used to send data to {Analytics}.
* *Red Hat or Satellite username*: This username is used to send data to {Analytics}.
* *Red Hat or Satellite password*: This password is used to send data to {Analytics}.
* *Last gather date for Automation Analytics*: Set the date and time
* *Automation Analytics Gather Interval*: Interval (in seconds) between data gathering.
. Click btn:[Save].

Reply:
Updated.